src/blocks/card.css

```diff
@@ -0,0 +1,95 @@
+
+.card {
+  box-sizing: border-box;
```

Comment:
you can move it to the * selector to set it for all elements

Reply:
Fix it 
![image](https://user-images.githubusercontent.com/97547824/211642908-0d1b296f-0bc1-4d76-84bf-ca0058501214.png)